stacks-signer/src/client/stacks_client.rs

```diff
@@ -1040,7 +1040,8 @@ mod tests {
     fn parse_valid_signer_slots_should_succeed() {
         let mock = MockServerClient::new();
         let clarity_value_hex =
-            "0x070b000000050c00000002096e756d2d736c6f7473010000000000000000000000000000000d067369676e6572051a8195196a9a7cf9c37cb13e1ed69a7bc047a84e050c00000002096e756d2d736c6f7473010000000000000000000000000000000d067369676e6572051a6505471146dcf722f0580911183f28bef30a8a890c00000002096e756d2d736c6f7473010000000000000000000000000000000d067369676e6572051a1d7f8e3936e5da5f32982cc47f31d7df9fb1b38a0c00000002096e756d2d736c6f7473010000000000000000000000000000000d067369676e6572051a126d1a814313c952e34c7840acec9211e1727fb80c00000002096e756d2d736c6f7473010000000000000000000000000000000d067369676e6572051a7374ea6bb39f2e8d3d334d62b9f302a977de339a";
+            "0x070b000000050c00000002096e756d2d736c6f7473010000000000000000000000000000000e067369676e6572051a8195196a9a7cf9c37cb13e1ed69a7bc047a84e050c00000002096e756d2d736c6f7473010000000000000000000000000000000e067369676e6572051a6505471146dcf722f0580911183f28bef30a8a890c00000002096e756d2d736c6f7473010000000000000000000000000000000e067369676e6572051a1d7f8e3936e5da5f32982cc47f31d7df9fb1b38a0c00000002096e756d2d736c6f7473010000000000000000000000000000000e067369676e6572051a126d1a814313c952e34c7840acec9211e1727fb80c00000002096e756d2d736c6f7473010000000000000000000000000000000e067369676e6572051a7374ea6bb39f2e8d3d334d62b9f302a977de339a";
```

Comment:
What's the delta here?

Reply:
Changing a hard-coded 13 to a 14 for `num-slots` in a type signature. Though I don't like that we use a hard-coded hex here because it obfuscates what's going on, and hex deserialization is not the purpose of this test.

I'll expand this into Rust code so that

1. We can use the appropriate constant instead of hard-coding a value and
2. Future deltas becomes easier to read.

Reply:
Done.